sdk/spring-sdk/src/test/resources/logback.xml

```diff
@@ -25,8 +25,8 @@
     <appender-ref ref="JSON-STDOUT"/>
   </appender>
 
-  <logger name="akka" level="INFO"/>
-  <logger name="kalix" level="INFO"/>
+  <logger name="akka" level="DEBUG"/>
+  <logger name="kalix" level="DEBUG"/>
```

Comment:
Did you actually want to change to debug? Or is it a leftover from troubleshooting?

Reply:
I don't remember to have changed this, but I think it's good to keep it like that for now. For instance, the generated proto descriptors are printed to the logs. It's good to have it during development. We can remove it later on.